psdk_wrapper/src/psdk_wrapper.cpp

```diff
@@ -25,8 +25,8 @@ PSDKWrapper::PSDKWrapper(const std::string &node_name)
     : rclcpp_lifecycle::LifecycleNode(
           node_name, "",
           rclcpp::NodeOptions().use_intra_process_comms(true).arguments(
-              {"--ros-args", "-r",
-               node_name + ":" + std::string("__node:=") + node_name}))
+              {"--ros-args", "-r", std::string("__ns:=/wrapper"), "--ros-args",
```

Comment:
do we need this anymore? Maybe we can remove the namespace line and hope it will be taken from the launch file.

Reply:
agreed. done here https://github.com/umdlife/psdk_ros2/pull/34/commits/91b8f7cbf432f15ff8a5178aefe7bbd5b6d7b1eb